docs/pages/en/4.providers/imagekit.md

```diff
@@ -18,3 +18,248 @@ export default {
+
+Custom coordinates can also be used to focus using parameter value `custom`.
+
+Moreover, ImageKit also provides smart cropping that can automatically detect most important part of the image using `auto`. And, `face` can be used to find out a face (or multiple faces) in an image and focus around that.
+
+Check out ImageKit's documentation on [focus](https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations#focus-fo) to learn more.
+
+
+### `blur`
+
+This can be used to blur an image. Use modifier `blur` to specify the Gaussian Blur radius that is to be applied on the image. Possible values include integers between `1` to `100`.
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/default-image.jpeg"
+  :modifiers="{ blur : 10}"
+/>
+```
+
+
+### `effectGray`
+
+Turn your image to a graycale version using `effectGray` modifier.
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/default-image.jpeg"
+  height="300"
+  :modifiers="{ effectGray : true}"
+/>
+```
+
+
+### `named`
+
+Use named transforms as an alias for an entire transformation string.
+
+For example, we can create a named transformation - `media_library_thumbnail` for a transformation string - `tr:w-100,h-100,c-at_max,fo-auto`.
+
+
+### `border`
+
+Add border to your images using modifier `border`. You can also set its width and color.
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/default-image.jpeg"
+  width="300"
+  :modifiers="{ border : '20_FF0000'}"
+/>
+```
+
+
+### `rotate`
+
+Use `rotate` modifier to rotate your image. Possible values are-  `0`, `90`, `180`, `270`, `360`, and `auto`.
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/default-image.jpeg"
+  :modifiers="{ rotate : 90 }"
+/>
+```
+
+
+### `radius`
+
+Give rounded corners to your image using `radius`. Possible values are- positive integers and `max`.
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/default-image.jpeg"
+  :modifiers="{ radius : 20}"
+/>
+```
+
+
+### `bg`
+
+Specify background color and its opacity for your image using `bg` modifier. 
+
+```html
+<nuxt-img
+  provider="imagekit"
+  src="/default-image.jpeg"
+  height="1200"
+  width="1200"
+  fit="pad_extract"
+  :modifiers="{ bg : '272B38' }"
+/>
+```
+
+
+### Read more about ImageKit crop, resize and other common transformations [here](https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations).
+
+## Overlay Transformation Modifiers
+Using ImageKit's Nuxt Image integration you can overlay images or text over other images for watermarking or creating a dynamic banner using custom text!
+
+### `overlayImage`
+
+Overlay an image on top of another image (base image) using `overlayImage` modifier. You can use this to create dynamic banners, watermarking etc.
+
```

Comment:
The overlayImage example was removed - I am assuming because there isn't another actual image available by default. However, we could still have a reference example like this:

```
<nuxt-img
  provider="imagekit"
  src="/default-image.jpg"
  :modifiers="{ overlayImage: 'some-overlay-image.jpg' }"
/>
```

Reply:
Another way to do this could be to overlay the same image:

https://ik.imagekit.io/nndev/tr:oi-default-image.jpg,ow-300,oh-200,ofo-top_left/default-image.jpg